frontend/src/modules/dashboard/components/dashboard-active-integrations.vue

```diff
@@ -47,7 +47,13 @@
             </p>
           </div>
           <div>
-            <div v-if="integration.status === 'done'">
+            <div
+              v-if="
+                integration.status === 'done'
+                  || (integration.status === 'error'
+                    && !isCurrentDateAfterGivenWorkingDays(integration.updatedAt, 3))
```

Comment:
i would have constand 3 days in one place so you can just change it in one place and it aligns on all places

Reply:
maybe you can also create function which is used on all places showIntegrationError or something and just manage logic there